common/tools/coin_info.py

```diff
@@ -398,32 +533,141 @@ def support_info(coins):
     if isinstance(coins, CoinsInfo):
         coins = coins.as_list()
```

Comment:
Specifying type in the line where we change the `coins` - `coins: Iterable[Coin] = ...` - unfortunately results in `Parameter declaration "coins" is obscured by a declaration of the same name` from `pyright` and `Name 'coins' already defined on line 521` from `mypy`

Reply:
in that case shouldn't `Coins = Iterable[Coin]` ?

Reply:
> in that case shouldn't `Coins = Iterable[Coin]` ?

That gives `Argument of type "Coins" cannot be assigned to parameter "__obj" of type "Sized" in function "len"` or `Cannot access member "append" for type "Coins"` in some places

 `Coins = Sequence[Coin]` is also complaining about missing `append` method

So seems like no easy solution :)

Reply:
> practically speaking, instead of `Coins`, accept an `Iterable[Coin]` for this function argument and then you can avoid the `list(values())` too

Done like this in [3f47380](https://github.com/trezor/trezor-firmware/pull/2286/commits/3f47380c5d9dde06c2d8ac13261a1a74b19ed7c8)